beacon-chain/core/helpers/rewards_penalties.go

```diff
@@ -83,3 +83,67 @@ func InactivityPenalty(
 	validatorBalance := EffectiveBalance(state, validatorIndex)
 	return baseReward + validatorBalance*epochsSinceFinality/params.BeaconConfig().InactivityPenaltyQuotient/2
 }
+
+// GetBalance returns the Gwei balance of a validator by its index
+//
+// Spec pseudocode definition:
+//   def get_balance(state: BeaconState, index: ValidatorIndex) -> Gwei:
+//     """
+//     Return the balance for a validator with the given ``index``.
+//     """
+//     return state.balances[index]
+func GetBalance(state *pb.BeaconState, validatorIndex uint64) uint64 {
```

Comment:
`validatorIndex` feels too verbose here, prefer `state.ValidatorBalances[idx]` over `state.ValidatorBalances[validatorIndex]`

```suggestion
func GetBalance(state *pb.BeaconState, idx uint64) uint64 {
```

Reply:
done

Reply:
thanks